src/views/help.vue

```diff
@@ -0,0 +1,255 @@
+<template>
+  <div class="help-container">
+    <div class="header">
+      <div class="title">Frequently Asked Questions</div>
+
+      <div class="description">
+        Your questions were not addressed in the questions below? Please contact our support via
+        <a target="_blank" :href="`https://api.whatsapp.com/send?phone=${phone}`">WhatsApp</a>
+        or email <b>suporte@weni.ai</b> <emoji name="Winking Face" />
+      </div>
+    </div>
+
+    <div class="content">
+      <div :style="{ height: 0 }">
+        <unnnic-accordion
+          v-for="(question, index) in questions"
+          v-model="question.open"
+          :key="index"
+          :title="question.title"
+          class="question"
+        >
+          <unnnic-button
+            v-if="question.video"
+            @click.stop="openVideo(question.video)"
+            slot="actions"
+            type="secondary"
+            size="small"
+            icon-left="button-play-1"
+          >
+            Watch the video
+          </unnnic-button>
+
+          <dynamic :template="`<span>${question.content}</span>`"></dynamic>
+        </unnnic-accordion>
+      </div>
+    </div>
+  </div>
+  
+</template>
+
+<script>
+import Vue from 'vue';
+import Emoji from '../components/Emoji.vue';
+
+const dynamic = {
+  props: ['template'],
+  components: {
+    Emoji,
+  },
+  data() {
+    return {
+      templateRender: null,
+    };
+  },
+  render(h) {
+    if (!this.templateRender) {
+      return h('div', 'loading...');
+    } else {
+      return this.templateRender();
+    }
+  },
+  watch: {
+    template:{
+      immediate: true,
+      handler() {
+        const res = Vue.compile(this.template);
+
+        this.templateRender = res.render;
+
+        this.$options.staticRenderFns = [];
+
+        this._staticTrees = []
+
+        for (let i in res.staticRenderFns) {
+          this.$options.staticRenderFns.push(res.staticRenderFns[i])
+        }
+      }
+    }
+  },
+};
+
+export default {
+  components: {
+    Emoji,
+    dynamic,
+  },
+
+  data() {
+    return {
+      phone: '5582900000000',
+
+      questions: [{
```

Comment:
Perhaps it would be better create this object in another file like on utils page.. so there u could create a FaqQuestions.js  and import it here

Reply:
Oh I see. In this specific case, this entire content is just for simulating, but in the final result those content will be in translations files.